it/pages/ChangeReturnFrequencyConfirmationISpec.scala

```diff
@@ -16,13 +16,13 @@
 
 package pages
 
-import play.api.i18n.Messages
+import play.api.i18n.{I18nSupport, Messages}
 import play.api.libs.ws.WSResponse
 import play.api.test.Helpers._
 import stubs.ContactPreferencesStub._
 import stubs.VatSubscriptionStub._
 
-class ChangeReturnFrequencyConfirmationISpec extends BasePageISpec {
+class ChangeReturnFrequencyConfirmationISpec extends BasePageISpec with I18nSupport {
```

Comment:
If you keep `implicit lazy val messages: Messages = Messages(Lang("en-GB"), messagesApi)` in BaseISpec you don't need this trait (and in the file below)